src/css/tabs/osd.less

```
@@ -307,6 +307,9 @@
         margin-top: 5px;
         margin-bottom: 8px;
         width: 100%;
+        .osd-elements-search-field {
```

Think OSD is duplicate in here
```suggestion
        .elements-search-field {
```

---

Changed